### ISSUE TITLE
[Accessibility][Keyboard Navigation]: Keyboard focus is not visible on 'Basic uncontrolled multi select example (allowFreeform: T, AutoComplete: T)' combobox list options.

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11199]
Split out of issue #6920

**Comment by xugao** [comment]
[Comment](https://microsoft-my.sharepoint-df.com/:x:/p/mgodbolt/EWp6C5th0sZMgOON89z0g4kBPi12LZ_vfhZQk6gfyc81nw?e=iu0nhL) from JSPURLIN:

For the most part I'm seeing focus be set on the selected item when the list is opened. The scaled example seems to have an issues (that's just a bug, not a specific accessibility bug).      The fact that focus doesn't go into the list is by design and aligns with the correct comboBox behavior (it is even described in the aria spec for comboBoxes). Additionally we have the comboBox marked up correctly so screen readers know how to understand the interactions of the comboBox. This is not a MAS bug

----

Xu: I will track the bug in virtualized/scaled example separately
